.gitignore

```diff
@@ -28,5 +28,6 @@ yarn-debug.log*
 yarn-error.log*
 
 .eslintcache
+cypress.config.local.ts
```

Comment:
Why did we ignore local/local-env? We should ignore prod ones for sure but the local ones are useful and shouldn't contain secrets.